CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# dbt_hubspot_source v0.17.0
+
+## Breaking Change (`--full-refresh` required after upgrading)  
+- Introduced a new `category` column to the below models. This association field differentiates records by either HUBSPOT_DEFINED (default label) or USER_DEFINED (custom label) and was introduced to the connector in October 2024.
```

Comment:
Based on these release notes should we be adding the `association_type` source table as a staging model?

This may not be directly in scope for addressing this issue. But maybe we can create a FR to gauge interest and add in the future if there's a need or want.

Reply:
Also while we're at it, let's create a separate FR to add more uniqueness tests to the staging models. This would be able to help catch any issues like this in the future and ensure data quality moving forward.

Reply:
good idea, added.
And I'll make those FRs!